features/support/messaging/et1_en.yml

```diff
@@ -1777,7 +1777,7 @@ en:
           ms: 'Ms'
         questions:
           full_name: 'Full name'
-          date_of_birth: 'Date of birth'
+          date_of_birth: 'Date of birth (optional)'
```

Comment:
Im not sure this change is correct - this is for the review section and you would not see the word 'optional' in there (intentionally anyway !)